library/tasks/TagSortBam.wdl

```diff
@@ -2,7 +2,7 @@ task CellSortBam {
   File bam_input
 
   # runtime values
-  String docker = "quay.io/humancellatlas/secondary-analysis-samtools:v0.2.2-1.6"
+  String docker = "quay.io/humancellatlas/secondary-analysis-sctools:ds-multi-tag-sort-170"
```

Comment:
You'll want to update this to a tagged version of master prior to merge, right?

Reply:
Definitely, will do